### ISSUE TITLE
[IMPROVEMENT] Change wordings node / disk tags to node / disk tags selector

### ISSUE THREAD

**Issue by a110605** [longhorn/longhorn#8913]
## Is your improvement request related to a feature? Please describe (👍 if you like this request)

Longhorn UI allows user to specify `node tag` and `disk tag` when creating volume or backing image. Longhorn will then make the copies of volume or backing image to the nodes which matches the node tags and disk tags. 

<img width="714" alt="Screenshot 2024-07-05 at 2 08 50 PM" src="https://github.com/longhorn/longhorn/assets/5744158/8167a829-b8f4-4044-931b-266b69159e5d">
<img width="509" alt="Screenshot 2024-07-05 at 2 09 02 PM" src="https://github.com/longhorn/longhorn/assets/5744158/ec7d37f0-a7c0-4a80-9748-8bba0e784e53">

In k8s yaml, it's specified in `parameter.nodeSelector` and `parameter.diskSelector` fields, see https://longhorn.io/docs/1.6.2/nodes-and-volumes/nodes/storage-tags/.

The wordings `node/disk tags` somehow not friendly for new user to understand the node/disk tags usage at first glimpse.

## Describe the solution you'd like
Propose to change the wordings from `node/disk tag` to `node/disk tag selector`. That's make it more match to its usage.

## Sub Tasks

Change the wordings in 
- [ ] backing image table and create backing image modal
- [ ] volume create modal and volume info details page

**Comment by houhoucoop** [comment]
I agree that consistency is good, but for non-technical or non-Longhorn professional users, `tag` might be clearer than `selector`?
